CRM/Export/BAO/ExportProcessor.php

```diff
@@ -1979,6 +1979,9 @@ public function mergeSameAddress() {
     $this->buildMasterCopyArray($sql);
 
     foreach ($this->contactsToMerge as $masterID => $values) {
+      if (strlen($values['addressee']) > 252) {
+        $values['addressee'] = substr($values['addressee'], 0, 252) . '...';
```

Comment:
I might use `CRM_Utils_String::ellipsify($values['addressee'], 255);`. It's what CRM_Core_DAO uses and it handles multibyte better. And then you also don't need the surrounding `if`.

Reply:
Well isn't that a handy little function?